sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj

```diff
@@ -5,12 +5,13 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
     <Description>Azure Data Factory V2 is the data integration platform that goes beyond Azure Data Factory V1's orchestration and batch-processing of time-series data, with a general purpose app model supporting modern data warehousing patterns and scenarios, lift-and-shift SSIS, and data-driven SaaS applications. Compose and manage reliable and secure data integration workflows at scale. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
-    <Version>4.10.0</Version>
+    <Version>4.12.0</Version>
```

Comment:
The last published version is 4.8.0, the next one is supposed to be 4.9.0.
Any special reason to skip to 4.12.0?

Reply:
The version was updated wrong by someone else, we have a different PR which fixed the issue, let's skip this PR, thanks!